WalletWasabi.Gui/Controls/QrCode.cs

```diff
@@ -41,7 +40,12 @@ public QrCode()
 
 			SaveCommand.ThrownExceptions
 				.ObserveOn(RxApp.TaskpoolScheduler)
-				.Subscribe(ex => Logger.LogError(ex));
+				.Subscribe(ex =>
+				{
+					// The error is thrown also in ReceiveTabViewModel -> SaveQRCodeCommand.ThrownExceptions.
+					// However we need to catch it here too but to avoid duplicate logging the following line commented out.
+					// Logger.LogWarning(ex);
+				});
```

Comment:
Do something with this.

Reply:
Actually it's somewhat acceptable and considering this holds back the release, I won't be nitpicking on this.